src/main/java/edu/greenblitz/robotName/subsystems/swerve/SwerveChassisUtils.java

```diff
@@ -1,23 +1,23 @@
 package edu.greenblitz.robotName.subsystems.swerve;
 
-import java.util.Scanner;
-import java.util.function.DoubleSupplier;
-
+import edu.greenblitz.robotName.subsystems.swerve.Chassis.ChassisConstants;
 public class SwerveChassisUtils {
+
+
+
     public static double joystickValueToLinearVelocity(double joystickValue, double velocityFactor) {
```

Comment:
Consider extracting the result of the duplicate ternary operation(that's the one with the ? and : ) to a variable to avoid code duplication and increase readability